src/test/java/org/apache/maven/plugins/ejb/EjbMojoTest.java

```diff
@@ -29,11 +29,19 @@
 import org.apache.maven.plugins.ejb.stub.MavenProjectResourcesStub;
 import org.apache.maven.plugins.ejb.utils.JarContentChecker;
 import org.apache.maven.project.MavenProject;
+import org.codehaus.plexus.util.FileUtils;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
 
 /**
  * EJB plugin Test Case
  */
-public class EjbMojoTest extends AbstractMojoTestCase {
+class EjbMojoTest extends AbstractMojoTestCase {
```

Comment:
`AbstractMojoTestCase` is based on JUnit 3 
We need migrate such test to `@MojoTest` annotation

Reply:
Yes I'll try migrate myself